crowdloans/rewards/polkadot.json

```diff
@@ -107,21 +108,32 @@
     },
     "parallelBonus": {
       "items": [
-        { "formula": "0.05", "logo": "/static/media/para.e1be6567.svg" }
+        { "formula": "0.10", "logo": "/static/media/para.e1be6567.svg" }
       ],
-      "tooltip": [{ "text": "Parallel Referral Bonus", "value": "5% PARA" }],
+      "tooltip": [{ "text": "Parallel Referral Bonus", "value": "10% PARA" }],
```

Comment:
Keep at 5% because it's 5% for referrer and referee. The total is 10% but we only show this in the homepage table.

Reply:
ok